app/util/concurrent/BackgroundProcessor.scala

```diff
@@ -36,8 +38,8 @@ object BackgroundProcessor {
 
   type SendType[T] = Tuple2[Option[Throwable], Option[T]]
   def send[PROC_RES,RESPONSE](cmd: BackgroundProcess[PROC_RES])(result: SendType[PROC_RES] => RESPONSE)(implicit mf: Manifest[PROC_RES]) = {
-    ask(ref, cmd)(cmd.timeout).mapTo[PROC_RES].asPromise.extend1 {
-      case Redeemed(v) => result(Tuple2(None, Some(v)))
+    ask(ref, cmd)(cmd.timeout).mapTo[PROC_RES].map {
```

Comment:
This is broken. Try to run an asset through intake

![screen shot 2015-02-01 at 5 09 11 pm](https://cloud.githubusercontent.com/assets/130194/5994097/3d69e0da-aa35-11e4-82b6-9b34d52db710.png)